examples/tutorial/tutorial_part_two.ipynb

```diff
@@ -33,7 +33,26 @@
     "key = \"\"\n",
     "# Set the SSH host\n",
     "host = \"\"\n",
-    "clipper = cm.Clipper(host, user, key)"
+    "clipper = cm.Clipper(host, user, key)\n",
+    "clipper.start()"
```

Comment:
Because we're moving cleanup to the end of part 2, there's no reason that clipper should need to be started again. Delete this line.